cli.js

```diff
@@ -32,29 +32,41 @@ if (args.help || args._.length === 0) {
   console.log("--ignore-module, -i   Won't tell you about module names passed in as --ignore-module / -i. Only usable with --unused")
   console.log('--entry               By default your main and bin entries from package.json will be parsed, but you can add more the list of entries by passing them in as --entry')
   console.log("--no-default-entries  Won't parse your main and bin entries from package.json will be parsed")
-  console.log('--detective           Pointing to a requireable path of a javascript file containing an alternative implementation of the detective module that supports extended or alternate syntaxes')
-  console.log("--extensions, -e      Comma-separated list of file extensions to use when resolving require paths. Eg. 'js,jsx'")
+  console.log('--detective           Requireable path containing an alternative implementation of the detective module that supports alternate syntaxes')
+  console.log("--extensions, -e      List of file extensions with detective to use when resolving require paths. Eg. 'js,jsx:detective-es6'")
```

Comment:
Yes, it's still supported, but I don't see any need to be overly documented about it. It sets the default, while most usage should be through the new syntax. 

You wouldn't need to specify anything to keep the existing `js` extension default. It only overrides the default, but I can find an additional example of multiple extensions working together. Not sure what it'd be right now though.